lib/ClangImporter/ImportDecl.cpp

```diff
@@ -1706,9 +1717,9 @@ namespace {
         Type underlyingType,
         ArrayRef<KnownProtocolKind> synthesizedProtocolAttrs,
         ArrayRef<ProtocolDecl *> protocols,
+        bool makeUnlabeledValueInit = false,
         Accessibility setterAccessibility = Accessibility::Private,
         bool isLet = true,
-        bool makeUnlabeledValueInit = false,
         bool isImplicit = true) {
```

Comment:
Moving defaulted parameters around like this is terrifying. We should really, really introduce an option set here.